### PR TITLE
changed geoLocation to Latitude and Longitude in schema

### PR DESCRIPTION
**Pull request by larsyaeger**

### CODE REVIEW

**Review by CorgiOnNeptune — APPROVED** [review]

File: prisma/schema.prisma [hunk 1]
```diff
@@ -19,7 +19,8 @@ model User {
   Like              Like[]
   currentSongUri    String    @map("current_song_uri")
```
Comment (Likai-L) [inline prisma/schema.prisma]:
```suggestion
  currentSongUri    String?    @map("current_song_uri")
```

File: prisma/schema.prisma [hunk 2]
```diff
@@ -19,7 +19,8 @@ model User {
...
   nowPlaying        Song      @relation(fields: [currentSongUri], references: [spotifySongUri])
```
Comment (Likai-L) [inline prisma/schema.prisma]:
```suggestion
  nowPlaying        Song?      @relation(fields: [currentSongUri], references: [spotifySongUri])
```

File: prisma/schema.prisma [hunk 3]
```diff
@@ -19,7 +19,8 @@ model User {
...
-  geoLocation       String    @map("geo_location")
+  Latitude          String
```
Comment (Likai-L) [inline prisma/schema.prisma]:
```suggestion
  Latitude          String?
```

File: prisma/schema.prisma [hunk 4]
```diff
@@ -19,7 +19,8 @@ model User {
...
+  Longitude         String
```
Comment (Likai-L) [inline prisma/schema.prisma]:
```suggestion
  Longitude         String?
```

**Review by Likai-L — CHANGES_REQUESTED** [review]

**Review by Likai-L — APPROVED** [review]
Good work, here's an avocado for you 🥑

**Pull request merged**